bode/bode/resources/task_relations/api.py

```diff
@@ -27,3 +31,15 @@ class TasksRelationsById(MethodView):
     @blueprint.response(200, SimpleTaskRelationSchema)
     def delete(self, relation_id):
         return TaskRelation.delete(relation_id)
+
+
+@blueprint.route("/<task_id>")
+class TasksInRelationWith(MethodView):
+    @blueprint.response(200, TasksRelationSchema(many=True))
+    def get(self, task_id):
+        try:
+            return TaskRelation.query.filter(TaskRelation.first_task_id == task_id).all() or abort(404)
+        except DataError:
+            abort(404)
+        except NoResultFound:
+            abort(404, message="Item not found.")
```

Comment:
Getting only the relations record is not really useful. When using this API route you'd want to have some data about the related tasks in the response, like `title`,  `due_date`, etc. The response scheme should more or less look like this:
```python
class RelatedTaskSchema(BaseSchema):
    id = fields.UUID(dump_only=True) # relation id
    type = fields.String() # relation type
    related_task = fields.nested(TaskSchema) # the task object that is related that is in relation with the given task
```

Also, you don't need to catch these errors when returning lists. A response with an empty list of related tasks is not exceptional and should return 200 response code.